.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recommendations": [
```

Comment:
https://dev.to/askrishnapravin/recommend-vs-code-extensions-to-your-future-teammates-4gkb